glide.yaml

```diff
@@ -91,7 +97,9 @@ import:
 # close-write carry
 - package: github.com/docker/docker
   repo:    https://github.com/openshift/moby-moby.git
-  version: openshift-3.10-docker-b68221c
+  # the commit below points to openshift-3.10-docker-b68221c branch
+  # note that we pin it to a specific commit because we intend to freeze dependencies as updating might introduce risk
+  version: fcdab531a6d9b133eaa3f2a3f6c11fd02a97de75
```

Comment:
did this imply actual changes before? That branches should be changed without a reason.